.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "tabWidth": 2,
```

Comment:
Can we change this to 4?

Reply:
yes, 2-space indent is a shitcoin 😂

Reply:
From my own defense, I copied that from another file 😉